plugins/modules/cloud/misc/proxmox_kvm.py

```diff
@@ -63,6 +63,12 @@
   bootdisk:
     description:
       - Enable booting from specified disk. C((ide|sata|scsi|virtio)\d+)
+  cipassword:
+    description:
+      - Cloud Init password of default user to create.
```

Comment:
```suggestion
      - Cloud Init password of default user to create.
    type: ???
    version_added: 1.2.0
```
Same for the other new options.

You also need a [changelog fragment](https://docs.ansible.com/ansible/devel/community/development_process.html#changelogs-how-to).

Reply:
Is there a method to determine the next version to be released?

Is there a type ipaddress or similar? Or should i just use string for the nameserver?

Reply:
You can find out about the next release here: #582 (1.1.0 will be released today, and this PR probably needs more review, so it won't make it into 1.1.0 with a high probability - so the next version it can get into will be 1.2.0)

There's no type `ipaddress`, you need to use `str`.

Reply:
:+1: thanks for your guidance!